lib/assets/javascripts/cartodb/table/overlays/map_overlays.js

```diff
@@ -603,11 +658,13 @@ cdb.admin.MapOverlays = cdb.core.View.extend({
 
   _setupCartoDBLogo: function(type, data) {
 
-    var overlay = this.cartodb_logo = new cdb.admin.overlays[type]({
+    var overlay = this.logo = new cdb.admin.overlays[type]({
```

Comment:
Yup, that's correct, its referring to the same thing. There's no other references to `cartodb_logo` in this file, I cleaned it up so that the property name matches the overlay type like all the others.